### PR TITLE
TEC-6465 modify company pojo to organization pojo

### PR DESCRIPTION
**Pull request by diepjy**
### Description of Changes
Adding an Organization POJO as per Linkedin V2 api  documentation.
Some code was added in the DefaultMapper to handle lists of URN. 
A new AddressV2 class exists as it seems that the address class has changed.
Added test. 

### Documentation
N/A

### Risks & Impacts
Because Linkedin's api documentation is not exactly clear,  the risk is to have confused the types of the different types, especially when it comes to urns, as those are represented by strings

### Testing
Unit tested the deserialization of the organization POJO

### Compare (For layered PRs)
to be merged after https://github.com/ebx/ebx-linkedin-sdk/pull/16
https://github.com/clementEchobox/ebx-linkedin-sdk/compare/TEC-6464-ConvertSTringToLinkedinURL2...TEC-6465-ModifyCompanyPOJOToOrganizationPojo

### CODE REVIEW

File: src/main/java/com/echobox/api/linkedin/jsonmapper/DefaultJsonMapper.java [hunk 1]
```diff
@@ -722,6 +721,9 @@ protected Object toJavaType(FieldWithAnnotation<LinkedIn> fieldWithAnnotation, J
     if (Map.class.equals(type)) {
       return convertJsonObjectToMap(rawValue.toString(), fieldWithAnnotation.getField());
     }
+    if (URN.class.equals(type)) {
```
Comment (eddspencer) [inline src/main/java/com/echobox/api/linkedin/jsonmapper/DefaultJsonMapper.java]: This already exists on line 694

File: src/main/java/com/echobox/api/linkedin/types/organization/Organization.java [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+   */
+  @Getter
+  @Setter
+  @LinkedIn
+  private String organizationType;
+  
+  @JsonMappingCompleted
+  protected void jsonMappingCompleted(JsonMapper jsonMapper) {
+
+    if (industriesRaw != null) {
```
Comment (eddspencer) [inline src/main/java/com/echobox/api/linkedin/types/organization/Organization.java]: Could this be done in the DefaultJsonMapper directly?

Comment (eddspencer) [reply src/main/java/com/echobox/api/linkedin/types/organization/Organization.java]: Adding:

```
      if (type.equals(URN.class)) {
        return (T) new URN(json);
      }
```

to DefualtJsonMapper:267 seems to work...I seem to remember Clement having this there as well. I am not sure this is exactly a neat solution but would be nice to have the logic centralised....there is already an exception at DefualtJsonMapper:179 so maybe we could add another here?

Comment (diepjy) [reply src/main/java/com/echobox/api/linkedin/types/organization/Organization.java]: Yeah, ideally we don't want to change the DefaultJsonMApper too much from the restFB impl incase there are breaking changes - we'd need to make sure we run a diff

**Pull request merged**